semgrep_output_v1.atd

```diff
@@ -787,6 +787,13 @@ type api_scans_findings <ocaml attr="deriving show"> = {
    cai_ids: string list;
 }
 
+type finding_hashes <ocaml attr="deriving show"> = {
+  start_line: string;
```

Comment:
would it be better to store this as an `int`?

Reply:
oooh. or maybe i misread, and this is the entire line itself.

Reply:
i might still recommend a `_hash` prefix after each of these fields, for dummies like me who may get confused :)

Reply:
Good call! Naming is hard